crates/forge/bin/cmd/debug.rs

```diff
@@ -25,33 +26,137 @@ pub struct DebugArgs {
     pub target_contract: Option<String>,
 
     /// The signature of the function you want to call in the contract, or raw calldata.
-    #[clap(long, short, default_value = "run()", value_name = "SIGNATURE")]
+    #[clap(
+        long,
+        short,
+        default_value = "run()",
+        value_parser = foundry_common::clap_helpers::strip_0x_prefix
+    )]
     pub sig: String,
 
-    /// Open the script in the debugger.
+    /// Max priority fee per gas for EIP1559 transactions.
+    #[clap(
+        long,
+        env = "ETH_PRIORITY_GAS_PRICE",
+        value_parser = parse_ether_value,
+        value_name = "PRICE"
+    )]
+    pub priority_gas_price: Option<U256>,
+
+    /// Use legacy transactions instead of EIP1559 ones.
+    ///
+    /// This is auto-enabled for common networks without EIP1559.
     #[clap(long)]
-    pub debug: bool,
+    pub legacy: bool,
+
+    /// Broadcasts the transactions.
+    #[clap(long)]
+    pub broadcast: bool,
+
+    /// Skips on-chain simulation.
+    #[clap(long)]
+    pub skip_simulation: bool,
+
+    /// Relative percentage to multiply gas estimates by.
+    #[clap(long, short, default_value = "130")]
+    pub gas_estimate_multiplier: u64,
```

Comment:
I'm having trouble understanding why changing all these fields is necessary in order to fix the referenced issue?